app/experimenter/legacy-ui/rapid/components/experiments/ResultsRow.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+
+import { SIGNIFICANCE_TIPS } from "experimenter-rapid/components/experiments/tooltips";
+
+enum Significance {
+  POSITIVE,
+  NEGATIVE,
+  NEUTRAL,
+}
+
+const showSignificanceField = (significance, interval) => {
+  {
+    let significanceIcon, className, testID;
+    switch (significance) {
+      case Significance.POSITIVE:
+        significanceIcon = (
+          <i
+            className="fas fa-arrow-up"
+            data-tip={SIGNIFICANCE_TIPS.POSITIVE}
+          />
+        );
+        className = testID = "positive-significance";
+        break;
+      case Significance.NEGATIVE:
+        significanceIcon = (
+          <i
+            className="fas fa-arrow-down"
+            data-tip={SIGNIFICANCE_TIPS.NEGATIVE}
+          />
+        );
+        className = testID = "negative-significance";
+        break;
+      case Significance.NEUTRAL:
+        significanceIcon = (
+          <i className="fas fa-minus" data-tip={SIGNIFICANCE_TIPS.NEUTRAL} />
+        );
+        className = testID = "neutral-significance";
```

Comment:
If `className` and `testID` are always the same can they be the same var?

Reply:
yea. I was trying to do that for clarity but clearly it's not more clear XD